src/components/Header/HeaderRightZone.js

```diff
@@ -0,0 +1,31 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import HeaderContext, { SLIM } from './HeaderContext';
+
+const propTypes = {
+    /** Classi addizionali per il componente HeaderRightZone */
+    className: PropTypes.string,
+};
+
+const defaultProps = {};
+
+class HeaderRightZone extends PureComponent {
+
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+    static contextType = HeaderContext;
+
+    render() {
+        const { className, ...attributes } = this.props;
+        const { type } = this.context;
+        const classes = classNames(className, {
+            'header-slim-right-zone': type === SLIM,
```

Comment:
Da Bootstrap Italia 1.3.6 è possibile usare `it-header-slim-right-zone`.

Reply:
Aggiunto anche la classe per BI >= 1.3.6